tests/zfs-tests/tests/functional/cli_root/zfs_copies/zfs_copies_001_pos.ksh

```diff
@@ -72,6 +72,7 @@ for val in 1 2 3; do
 	log_must $ZFS create -o copies=$val $fs1
 	if is_global_zone; then
 		log_must $ZFS create -V $VOLSIZE -o copies=$val $vol1
+		block_device_wait
```

Comment:
It wouldn't hurt but the way `block_device_wait` is implemented it can't fail so it doesn't really buy us anything.  For consistency with the other places this is used I opted not to use log_must.